wger/core/views/user.py

```diff
@@ -323,6 +330,82 @@ def preferences(request):
         return render(request, 'user/preferences.html', template_data)
 
 
+@login_required
+def sync_fitbit(request):
+    """Authorizes the connection with Wger
+
+    """
+
+    if settings.WGER_SETTINGS['FITBIT_CLIENT_ID'] is None:
```

Comment:
Impressive work! 🙌🏼 You could consider testing for the availability of a secret key too.

Reply:
> Impressive work! 🙌🏼 You could consider testing for the availability of a secret key too.

Working on it